src/EnergyPlus/LowTempRadiantSystem.cc

```diff
@@ -5130,7 +5130,6 @@ namespace LowTempRadiantSystem {
         using DataHVACGlobals::TimeStepSys;
 
         // SUBROUTINE PARAMETER DEFINITIONS:
-        auto constexpr RoutineName("UpdateLowTempRadiantSystem");
```

Comment:
These RoutineName-s are typicaally used in error strings, but if this routine does not have error messages, is it necessary to keep them?